components/Modals/Modal.tsx

```diff
@@ -2,30 +2,47 @@ import styled, { ThemeContext } from 'styled-components'
 import { useSpring, animated } from 'react-spring'
 import { MdClose } from 'react-icons/md'
 import { ModalProps } from '../../lib/gymmieInterfaces'
-import { centeredItems } from '../styles/HelperStyles'
+import { borderRadius, centeredItems } from '../styles/HelperStyles'
+import { SmallFormButton } from '../styles/ButtonStyle'
 
 export default function Modal({
   modalIsOpen,
-  modalType,
   modalText,
   closeModal,
-}: ModalProps) {
+  modalContinueButtonFn,
+}: Partial<ModalProps>) {
   const animation = useSpring({
     config: {
       duration: 250,
     },
-    opacity: modalIsOpen ? 1 : 0,
+    opacity: 1,
     transform: modalIsOpen ? `translateY(0%)` : `translateY(-100%)`,
   })
 
   return (
     <>
       {modalIsOpen && (
-        <Background modalType={modalType}>
+        <Background>
           <animated.div style={animation}>
             <ModalWrapper>
               <ModalContent>{modalText}</ModalContent>
-              <CloseModalButton aria-label="Close modal" onClick={closeModal} />
+              <ModalButtonWrapper>
+                <ModalButton type="button" onClick={closeModal}>
+                  Cancel
+                </ModalButton>
+                <ModalButton
+                  type="button"
```

Comment:
De default type for a `button` is already `type=button` so it's actually not necessary (but up to you if you want to be explicit about it)